configure.ac

```diff
@@ -2611,6 +2640,13 @@ AC_SUBST(TCLLINK)
 # Determine which languages to use for examples/test-suite
 #----------------------------------------------------------------
 
+SKIP_C=
+if test -x "$CC" || test -z "$CXX" ; then
```

Comment:
Should this `test -x` be `test -z`?

`-x` seems wrong here as this currently seems like it will skip C testing if `$CC` is executable, which I'd expect it always will be (since if we don't have a working C compiler, configure should bail out with an error).

I checked the last CI run and we actually get away with this in CI because `CC` doesn't have a full path there, and `test -x gcc` is not true because `test -x` doesn't try to resolve via `PATH`.  I do think it's wrong though.

Reply:
This definitely seems wrong, thanks for noticing. Apparently it always was like this, ever since a13c5976b (Fixed naming convention issues (internal Swig variables, shadow -> proxy, test.c -> runme.c, etc.), modified Makefiles, bugfixes ("name" -> "sym:name"). Started C test-suite., 2008-07-09), but this doesn't mean anything, of course.

AFAICS this entire fragment could be just removed completely because we require both C and C++ compiler.

Reply:
> AFAICS this entire fragment could be just removed completely because we require both C and C++ compiler.

Indeed - I've merged master to the branch (no manual intervention needed) and have removed the check - `SKIP_C` is now always empty (but still present in case we later grow dependencies here, e.g. we might conceivably start using some C test harness in the future).